io-engine/src/bdev/nexus/mod.rs

```diff
@@ -123,12 +123,16 @@ pub fn register_module() {
 
 /// called during shutdown so that all nexus children are in Destroying state
 /// so that a possible remove event from SPDK also results in bdev removal
-pub async fn nexus_children_to_destroying_state() {
-    info!("Setting all nexus children to destroying state...");
+pub async fn shutdown_nexuses() {
```

Comment:
How does this work, if we're only destroying the children, is this "graceful"?
what if there's ongoing IO?